### ISSUE TITLE
Dominance ?

### ISSUE THREAD

**Issue by onyalcin** [albertobeta/AffectiveSlider#5]
Is there any attempt to add dominance dimension in your scale?

**Comment by albertobeta** [comment]
No. [See the paper](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0148037).

> Contrary to SAM, the AS does not measure dominance. We excluded this dimension for two main reasons: first, Russel’s bipolar emotional space constitutes the, so called, “core affect” and is sufficient alone to measure basic emotion [21]. As a result, dominance can be considered redundant and seen as a consequence of core affect. Second, this dimension has not shown consistent effects across studies
